graders/c/requirements.txt

```diff
@@ -0,0 +1,4 @@
+matplotlib==3.3.4
+numpy==1.20.1
+Pillow==8.1.0
+sympy==1.7.1
```

Comment:
Out of curiosity, why are these required at all? I don't see them used in `cgrader.py`.

Reply:
They are not used in cgrader itself, but:
* matplotlib is used in https://github.com/PrairieLearn/PrairieLearn/blob/7a7647fe1b6624f7388e64f6b4969ba016d93dfc/exampleCourse/questions/demo/autograder/c/drawTriangle/tests/test.py#L5
* sympy and numpy are used in some of the questions I originally used this grader for, so they're kept for compatibility with those (plus any other production questions that may still depend on them);
* Pillow is not directly used, but sympy depends on it, and I've had problems with the new version of Pillow on pip3 (it has some undisclosed dependencies). Version 8.1.0 does not cause these issues.

Reply:
Ah, ok.

I think we need a better house view on things like this - my expectation would be that if specific courses need additional libraries, they should make their own image that installs anything extra they may need. Otherwise these core images keep getting fatter and fatter.

All that said, I acknowledge that we _don't_ have a well defined house view like that and that maintaining backwards-compat is important, so this looks fine!